src/cnpj/validation.ts

```diff
@@ -0,0 +1,70 @@
+export function validateCNPJ(cnpjReceveid: string | number) {
+  let cnpj = String(cnpjReceveid).replace(/[&\/\\#,+()$~%.'":*?<>{}-]/g, '')
```

Comment:
eslint is throwing an error because of unnecessary escape character
please ensure that you have prettier and eslint working in the project
```suggestion
  let cnpj = String(cnpjReceveid).replace(/[&/\\#,+()$~%.'":*?<>{}-]/g, '')
```